### DISCUSSION TITLE
Could the chore "release" PRs be updated to consistently print the version in the title?

### DISCUSSION THREAD

**Discussion by Nantris** [ueberdosis/tiptap#5400] [General Questions] [answered]
I'm trying to bisect a bug that was introduced between 2.5.4 and 2.5.5 and noticed that [one commit features the version](https://github.com/ueberdosis/tiptap/commit/46079c28a69acac3f70b811c7e3ff49ddd40e645) and [the other doesn't](https://github.com/ueberdosis/tiptap/commit/22cb0389973fb52ca279634915374fd46ddf7724). It would be helpful in these cases if all of these releases showed their version in the title.

**Comment by nperez0111** [comment, marked as answer]
It is manually done right now so hard to keep it consistent. But the git tags should be the give away on the version number

**Reply by Nantris** [reply]
The issue is more related to when viewing them in the list view, but I understand standardization will have to wait until it's automated. Thanks for your reply @nperez0111!

![image](https://github.com/user-attachments/assets/210efe2d-b25e-47ad-85ae-2bb10f83dbef)
